kubernetes/rucio/README.md

```diff
@@ -0,0 +1,146 @@
+Adapted from rucio instructions here: https://github.com/rucio/helm-charts/tree/master/rucio-server
+
+
+# First time setup
+
+## Setup a new cluster in the CMSRucio project:
+
+
+    openstack coe cluster delete --os-project-name CMSRucio  cmsruciotest
+    openstack coe cluster create cmsruciotest --keypair lxplus  --os-project-name CMSRucio   --cluster-template kubernetes-preview --node-count 4
+    openstack coe cluster list --os-project-name CMSRucio # Monitor creation status
+
+
+### If setting up a new/changed cluster:
+
+
+    rm key.pem cert.pem ca.pem config
+    openstack coe cluster config  --os-project-name CMSRucio  cmsruciotest
+    export KUBECONFIG=/afs/cern.ch/user/e/ewv/config
+
+
+Copy and paste the last line. On subsequent logins it is all that is needed. Now make sure that your nodes exist:
+
+    -bash-4.2$ export KUBECONFIG=/afs/cern.ch/user/e/ewv/config
+    -bash-4.2$ kubectl get nodes
+    NAME                                 STATUS    ROLES     AGE       VERSION
+    cmsruciotest-mzvha4weztri-minion-0   Ready     <none>    5m        v1.11.2
+    cmsruciotest-mzvha4weztri-minion-1   Ready     <none>    6m        v1.11.2
+    cmsruciotest-mzvha4weztri-minion-2   Ready     <none>    6m        v1.11.2
+    cmsruciotest-mzvha4weztri-minion-3   Ready     <none>    6m        v1.11.2
+
+## Install helm if needed
+
+    mkdir $HOME/bin
+    export PATH=$PATH:$HOME/bin
+    cp helm $HOME/bin
+
+## Setup helm if needed
+
+    export KUBECONFIG=/afs/cern.ch/user/e/ewv/config
+    kubectl config current-context
+    helm repo add rucio https://rucio.github.io/helm-charts
+
+## Label nodes for ingress (pick from above list) and add the same nodes to the DNS registration
+
+    kubectl label node cmsruciotest-mzvha4weztri-minion-0 role=ingress
```

Comment:
here users need to know where node name comes from, I know, but it is confusing. A simple comment here would be appropriate that node name came from output of previously executed `kubectl get nodes`  command.